src/ert/config/parsing/config_schema_deprecations.py

```diff
@@ -196,17 +196,14 @@
     ),
     DeprecationInfo(
         keyword="QUEUE_OPTION",
-        message="QUEUE_QUERY_TIMEOUT as QUEUE_OPTION to the TORQUE system will be ignored "
-        "when using the scheduler, and it is not recommended to use this QUEUE_OPTION. "
-        "It has been used in the past to set the time ERT will wait before giving "
-        "up on hanging backend (TORQUE/PBS) when submitting jobs or job status querying.",
+        message="QUEUE_QUERY_TIMEOUT as QUEUE_OPTION is ignored. "
+        "Please remove the line.",
```

Comment:
Nice cleanup! This should probably have been in the commit removing the legacy torque driver :)